src/systems/user_commands/UserCommands.cc

```diff
@@ -356,8 +360,13 @@ class WheelSlipCommand : public UserCommandBase
               const msgs::WheelSlipParametersCmd &_b)
             {
               return
-                _a.model_name() == _b.model_name() &&
-                _a.link_name() == _b.link_name() &&
+                (
+                  (_a.entity().id() != kNullEntity && _a.entity().id() == _b.entity().id()) ||
+                  (
+                    _a.entity().name() == _b.entity().name() &&
+                    _a.entity().type() == _b.entity().type()
+                  )
```

Comment:
I'm not sure, but maybe entity ids are unique, and this "or" section is redundant; i.e. maybe having the same id implies the name and types match ?

Reply:
`id == 0` is used when you don't know the id and you want to provide a name instead.